ruff_lsp/server.py

```diff
@@ -935,8 +962,10 @@ def document_from_kind(uri: str, kind: str) -> Document:
 
     actions: list[CodeAction] = []
 
-    # Add "Ruff: Autofix" for every fixable diagnostic.
-    if settings.get("codeAction", {}).get("fixViolation", {}).get("enable", True):
+    # If the linter is enabled, add "Ruff: Autofix" for every fixable diagnostic.
+    if settings.get("codeAction", {}).get("fixViolation", {}).get(
+        "enable", True
+    ) and lint_enable(settings):
```

Comment:
nit: slight preference to have `lint_enable` first which then reads as "if lint is enabled, and if this code action is enabled, ..."

Reply:
Agreed, thanks.